rubi/docs/source/conf.py

```diff
@@ -12,9 +12,9 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
 project = 'rubi'
-copyright = '2022, denver'
-author = 'denver'
-release = '0.4.2'
+copyright = '2022, Rubicon'
+author = 'Rubicon'
+release = '2.0.0'
```

Comment:
i think going forward we will want to have a `change log` and `release notes` for new versions. we can land on a semantic versioning later, for now lets continue to iterate towards a stable release (tests/data/other bells and whistles)